envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/resource/listeners/filters/rbac/RBACFilterFactoryJwtTest.kt

```diff
@@ -310,6 +385,13 @@ internal class RBACFilterFactoryJwtTest : RBACFilterFactoryTestUtils {
         }
     """
 
+    private fun withNegatedPrincipal(principal: String, negatedPrincipal: String) = """{
```

Comment:
`with` does not say anything to me without looking for function usage

Reply:
changed